pages/common/ghost.md

```diff
@@ -0,0 +1,36 @@
+# ghost
+
+> A blogging platform and headless CMS.
+> More information: <https://ghost.org>.
+
+- Install ghost in the current directory:
+
+`ghost install`
+
+- Start an instance of Ghost:
+
+`ghost start`
+
+- Restart the Ghost instance:
+
+`ghost restart`
+
+- Check the system for any potential hiccups when on install or update of Ghost:
+
+`ghost doctor`
+
+- View the logs of a Ghost instance:
+
+`ghost log [name]`
+
+- Run a Ghost instance directly (used by process managers and for debugging):
+
+`ghost run`
+
+- View running ghost processes:
+
+`ghost ls`
+
+- View or edit Ghost configuration:
+
+`ghost config [key] [value]`
```

Comment:
```suggestion
`ghost config [key] [value]`
```
Same as above. Although I wouldn't mesh up both viewing and editing the configuration, it's confusing.

Reply:
Do you mean this @zdroid?
```suggestion
`ghost config {{key}} {{value}}`
```